website/docs/segments/gitversion.mdx

```diff
@@ -0,0 +1,50 @@
+---
+id: gitversion
+title: GitVersion
+sidebar_label: GitVersion
+---
+
+## What
+
+Display the [GitVersion][gitversion] version.
+I _strongly_ recommend using [GitVersion Portable][gitversion-portable] for this.
+
+:::caution
+The GitVersion CLI can be a bit slow causing the prompt to feel slow.
```

Comment:
I'm wondering if there's a way to cache the information. Maybe by folder and interval?

Reply:
Good thought, do you have any example of how this is done in another segment? I tried to implement it a bit like NBGV is implemented since it's quite similar.

Reply:
@Lemorz56 there's a good pointer [here](https://github.com/JanDeDobbeleer/oh-my-posh/blob/c93d637128b47bd235776a4833abb59da28d44b2/src/http/request.go#L28).

Reply:
I've tried a lot with testing both via the tests but also trying to install and test it manually, but I can't really figure out what is wrong with the way I implemented the cache. Ive pushed it and I will continue to debug and see where my misstake is, but if you have the time and can, i'd love if you can take a look at the code too 😄